tenzir/integration/data/reference/functions/test_otherwise/step_00.ref

```diff
@@ -0,0 +1,29 @@
+{"x": 1, "y": -1, "z": 1}
+{"x": 2, "y": -2, "z": 2}
+{"z": null}
+{"x": 4, "y": -4, "z": 4}
+{"x": "5", "y": -5, "z": "5"}
+{"y": -6, "z": -6}
+warning: field `x` not found
+ --> <input>:1:17
+  |
+1 | read_json | z = x.otherwise(y)
+  |                 ~ 
+  |
```

Comment:
Is this the behavior that we want, getting a warning here? I'm aware that it would be quite unusual for a function to circumvent the normal rules here.

Reply:
We'll keep this warning around for now, knowing that it is undesired in many scenarios.